geomesa-kafka/geomesa-kafka-08-datastore/src/test/scala/org/locationtech/geomesa/kafka08/consumer/KafkaConsumerTest.scala

```diff
@@ -118,11 +120,10 @@ class KafkaConsumerTest extends Specification with HasEmbeddedKafka {
       for (i <- 0 until 10) {
         messages(i) mustEqual i.toString
       }
-
       success
     }
 
-    "read messages from various offsets" >> {
+    "read meZNRecordSerializerssages from various offsets" >> {
```

Comment:
The way "messages" got spliced leads me to believe that there was a search/replace command that went wild, yes?